src/sharding/Shard.js

```diff
@@ -132,15 +132,15 @@ class Shard extends EventEmitter {
      * @event Shard#spawn
      * @param {ChildProcess|Worker} process Child process/worker that was created
      */
-    this.emit('spawn', this.process || this.worker);
+    this.emit('spawn', this.process ?? this.worker);
 
-    if (spawnTimeout === -1 || spawnTimeout === Infinity) return this.process || this.worker;
+    if (spawnTimeout === -1 || spawnTimeout === Infinity) return this.process ?? this.worker;
     await new Promise((resolve, reject) => {
       const cleanup = () => {
         clearTimeout(spawnTimeoutTimer);
-        this.off('ready', onReady);
-        this.off('disconnect', onDisconnect);
-        this.off('death', onDeath);
+        this.removeListener('ready', onReady)
+          .removeListener('disconnect', onDisconnect)
+          .removeListener('death', onDeath);
```

Comment:
nit: Really no need to rename `off` from `removeListener` since they're the same, but its fine!

Reply:
If that were so, I'd believe it was for old Node.js compatibility (`off` is rather a new feature) and/or consistency.

Since this PR changes everything, I believe it's right to change the remaining `removeListener` to `off`, for the same reason we use `on` over `addListener` (or was it `createListener`?)

Reply:
the reason crawl gave at the time was incase it was ever removed, [see discord conversation](https://discord.com/channels/222078108977594368/222197033908436994/682992005348982811)

Reply:
Yeah, I think we can safely use `off` by now though, it's safe to stay.